### PR TITLE
Minor fixes for wellness

### PR DESCRIPTION
**Pull request by Andati**

### CODE REVIEW

File: opensrp-reporting/src/main/java/org/smartregister/reporting/dao/ReportIndicatorDaoImpl.java [hunk 1]
```diff
@@ -155,8 +155,8 @@ private int executeQueryAndReturnCount(String queryString, String date, SQLiteDa
         // Use date in querying if specified
         String query = "";
         if (date != null) {
-            Timber.i("QUERY : %s", queryString);
-            query = queryString.contains("'%s'") ? String.format(queryString, date) : queryString;
+            query = queryString.contains("%s") ? queryString.replaceAll("%s", date.split(" ")[0]) : queryString;
```
Comment (ekigamba) [inline opensrp-reporting/src/main/java/org/smartregister/reporting/dao/ReportIndicatorDaoImpl.java]: Make this date optional

File: opensrp-reporting/src/main/java/org/smartregister/reporting/ReportingLibrary.java [hunk 2]
```diff
@@ -198,12 +198,16 @@ private void readConfigFile(String configFilePath, SQLiteDatabase sqLiteDatabase
...
+
+                    String query = indicatorYamlConfigItem.getIndicatorQuery();
+                    if(!query.trim().isEmpty()) {
+                        indicatorQuery = new IndicatorQuery(null, indicatorYamlConfigItem.getKey()
+                                , indicatorYamlConfigItem.getIndicatorQuery()
+                                , 0
+                                , indicatorYamlConfigItem.isMultiResult());
+                        indicatorQueries.add(indicatorQuery);
+                    }
                     reportIndicators.add(indicator);
```
[the remark this answers is not in the document]
Comment (Andati) [reply opensrp-reporting/src/main/java/org/smartregister/reporting/ReportingLibrary.java]: I need the code whose count is then manually inputted in monthly tallies form before submitting

File: opensrp-reporting/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -5,10 +5,11 @@
     <application
         android:allowBackup="true"
         android:label="@string/app_name"
-        android:supportsRtl="true">
+        android:supportsRtl="true"
+        android:theme="@style/AppTheme">
```
[the remark this answers is not in the document]
Comment (Andati) [reply opensrp-reporting/src/main/AndroidManifest.xml]: sure

File: opensrp-reporting/src/main/java/org/smartregister/reporting/ReportingLibrary.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (ekigamba) [reply opensrp-reporting/src/main/java/org/smartregister/reporting/ReportingLibrary.java]: Ooo! Poa

**Pull request merged**